openedx/core/djangoapps/content/block_structure/migrations/0003_blockstructuremodel_storage.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, unicode_literals
 
+from __future__ import absolute_import
```

Comment:
It looks like all of these migrations files that got changed already had the `from __future__ import absolute_import` import, could you remove the duplicate lines in these files?

Reply:
Removed duplicate import from migrations